test/functional/plugin/lsp_spec.lua

```diff
@@ -241,7 +241,7 @@ describe('LSP', function()
 
     it('should succeed with manual shutdown', function()
       local expected_callbacks = {
-        {NIL, "shutdown", {}, 1};
+        {NIL, "shutdown", {}, 1, NIL};
```

Comment:
Since this `shutdown` is called as a request, bufnr is added to the callback argument this time.
`test` is called as notify, so there is no change in the callback argument.